<launch>
  <include file="$(find-pkg-share autoware_diagnostic_graph_aggregator)/launch/aggregator.launch.xml">
    <arg name="graph_file" value="$(find-pkg-share autoware_diagnostic_graph_aggregator)/example/graph/main.yaml"/>
  </include>
  <executable cmd="python3 $(find-pkg-share autoware_diagnostic_graph_aggregator)/example/dummy-diags.py"/>
</launch>
